trip-to-carbon.ts

```diff
@@ -71,4 +74,4 @@ function transformCountryOption(countryOption: string) {
     default:
       return 'def'
   }
-}
\ No newline at end of file
+}
```

Comment:
Why did this change?

Reply:
Not sure why that changed, the only difference I could spot was a new line in my commit at the end of the file.